cmd/bootstrap/permissionless-test/permissionless-collector-test.sh

```diff
@@ -0,0 +1,183 @@
+  for node in $permissionless_collectors
+  do
+    cp "./$clusteringpath" "./permissionless/$node/$clusteringpath"
+  done
+fi
+
+# cluster voting
+for node in $permissionless_collectors
+do
+  $transitcmd generate-cluster-block-vote -b "./permissionless/$node"
+done
+
+# collectors push votes, and bootstrapping machine retrieves them
+if [ -n "$bucket" ]
+then
+  # collectors push using transit script
+  for node in $permissionless_collectors
+  do
+    $transitcmd push-cluster-block-vote -g "$bucket" -t "$token" -b "./permissionless/$node"
+  done
+  gsutil cp "gs://$bucket/$token/root-cluster-block-vote.*" "./$clustervotespath/"
+else
+  # copy locally
+  for node in $permissionless_collectors
+  do
+    cp "./permissionless/$node/private-root-information/private-node-info_$node/root-cluster-block-vote.json" "./$clustervotespath/root-cluster-block-vote.$node.json"
+  done
+fi
+
+
+# root block creation should succeed now that we have enough votes
+$bootstrapcmd rootblock \
+    --root-chain bench \
+    --root-height 0 \
+    --root-parent 0000000000000000000000000000000000000000000000000000000000000000 \
+    --root-view 0 \
+    --epoch-counter 0 \
+    --epoch-length 30000 \
+    --epoch-staking-phase-length 20000 \
+    --epoch-dkg-phase-length 2000 \
+    --random-seed 00000000000000000000000000000000000000000000000000000000deadbeef \
+    --collection-clusters $clusterCount \
+    --use-default-epoch-timing \
+    --kvstore-finalization-safety-threshold=1000 \
+    --kvstore-epoch-extension-view-count=2000 \
+    --config ./node-config.json \
+    -o ./ \
+    --partner-dir "$partner_dir" \
+    --partner-weights ./partner-weights.json \
+    --internal-priv-dir "$keygen_dir/private-root-information" \
+    --intermediary-clustering-data "./$clusteringpath" \
+    --cluster-votes-dir "./$clustervotespath"
+
+
+# root block finalization should succeed with enough consensus votes
+$bootstrapcmd finalize \
+    --config ./node-config.json \
+    --partner-dir "./$partner_dir" \
+    --partner-weights ./partner-weights.json \
+    --internal-priv-dir "$keygen_dir/private-root-information" \
+    --dkg-data ./private-root-information/root-dkg-data.priv.json \
+    --root-block ./public-root-information/root-block.json \
+    --intermediary-bootstrapping-data ./public-root-information/intermediary-bootstrapping-data.json \
+    --root-block-votes-dir ./public-root-information/root-block-votes/ \
+    --root-commit 0000000000000000000000000000000000000000000000000000000000000000 \
+    --genesis-token-supply="1000000000.0" \
+    --service-account-public-key-json "{\"PublicKey\":\"R7MTEDdLclRLrj2MI1hcp4ucgRTpR15PCHAWLM5nks6Y3H7+PGkfZTP2di2jbITooWO4DD1yqaBSAVK8iQ6i0A==\",\"SignAlgo\":2,\"HashAlgo\":1,\"SeqNumber\":0,\"Weight\":1000}" \
+    -o ./
+
+#TODO: clean up files
+#rm -rf ./permissionless ./public-root-information ./private-root-information ./execution-state "$keygen_dir" "$partner_dir"
+#rm -f node-config.json partner-weights.json
```

Comment:
Could you add a gitignore entry that will ignore all files in this directory besides the shell script?